test/gateway/eip20_cogateway/progress_mint.js

```diff
@@ -0,0 +1,243 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const BN = require('bn.js'),
+    Utils = require("../../test_lib/utils"),
+    EIP20CoGatewayHelper = require("./helpers/helper");
+    EIP20CoGateway = artifacts.require('EIP20CoGateway'),
+    MockSafeCore = artifacts.require('MockSafeCore'),
+    MessageBus = artifacts.require('MessageBus'),
+    UtilityToken = artifacts.require('UtilityToken'),
+    EIP20Token = artifacts.require('EIP20Token'),
+    TestEIP20CoGateway = artifacts.require('TestEIP20CoGateway'),
+    MockUtilityToken = artifacts.require('MockUtilityToken');
+
+let eip20CoGateway,
+    stateRoot = "0x70b4172eb30c495bf20b5b12224cd2380fccdd7ffa2292416b9dbdfc8511585d",
+    valueToken,
+    mockSafeCore,
+    organization,
+    gateway,
+    utilityToken,
+    bountyAmount,
+    owner,
+    staker,
+    stakerBalance,
+    rewardAmount,
+    symbol = "OST",
+    name = "Simple Token",
+    decimals = 18,
+    helper;
```

Comment:
The length of these declarations looks unhealthy. Did we mess up with the architecture?

Reply:
Do you have any strong opinion against this.Is it fine to keep these declarations as of now ?
This makes writing the tests easy.